docs/int/openapi.yaml

```diff
@@ -796,6 +796,196 @@ paths:
               schema:
                 $ref: '#/components/schemas/Collection'
 
+  /api/metadata/job-codes:
+    get:
+      summary: Get all Job Codes
+      description: Returns all Job Codes
+      tags:
+        - MetaData
+      parameters:
+        - $ref: '#/components/parameters/PaginationSizeQueryRequired'
+        - $ref: '#/components/parameters/PaginationFromQueryRequired'
+        - $ref: '#/components/parameters/SortOrderQuery'
+      responses:
+        '200':
+          description: OK
+          headers:
+            X-Total-Count:
+              $ref: '#/components/headers/XTotalCount'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/JobCode'
+    post:
+      summary: Define a Job Code
+      description: Create a new Job Code in OSMT
+      tags:
+        - MetaData
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/JobCode'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/JobCode'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+
+  /api/metadata/job-codes/{id}:
+    parameters:
+      - $ref: '#/components/parameters/IdPath'
+    get:
+      summary: Retrieve a Job Code
+      description: Retrieve all the information about a Job Code
+      tags:
+        - MetaData
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JobCode'
+    delete:
```

Comment:
- [ ] Please make this /api/metadata/jobcodes/{id}/remove for API consistency

with the Collections delete endpoint. We will remove verbs from all API routes in API v3.0.